pkg/controllers/routing/network_routes_controller.go

```diff
@@ -43,12 +44,14 @@ const (
 	nodeCustomImportRejectAnnotation = "kube-router.io/node.bgp.customimportreject"
 	pathPrependASNAnnotation         = "kube-router.io/path-prepend.as"
 	pathPrependRepeatNAnnotation     = "kube-router.io/path-prepend.repeat-n"
-	peerASNAnnotation                = "kube-router.io/peer.asns"
-	peerIPAnnotation                 = "kube-router.io/peer.ips"
-	peerLocalIPAnnotation            = "kube-router.io/peer.localips"
+
+	peerASNAnnotation     = "kube-router.io/peer.asns"
```

Comment:
We should probably put a comment above these saying that they're deprecated or some such so that we don't have to carry them indefinitely.

We should also update the markdown docs with the same message

Reply:
Updated the markdown docs with the new annotation + examples.